share/module_check_a_mundo.F

```diff
@@ -994,18 +994,18 @@ SUBROUTINE check_nml_consistency
 !-----------------------------------------------------------------------
 
       oops = 0
-      DO i = 1, model_config_rec % max_dom
-         IF ( model_config_rec%bl_mynn_edmf(i) .EQ. MYNN_STEM_EDMF .OR. &
-              model_config_rec%bl_mynn_edmf(i) .EQ. MYNN_TEMF_EDMF) THEN
-               model_config_rec%shcu_physics(i) = 0  ! maxdom
-               model_config_rec%ishallow = 0         ! not maxdom
-               oops = oops + 1
+      EDMFMAX = MAXVAL(model_config_rec%bl_mynn_edmf(1:model_config_rec%max_dom))
+      SCHUMAX = MAXVAL(model_config_rec%shcu_physics(1:model_config_rec%max_dom))
+         IF ( ( ( EDMFMAX .GT. 0 ) .AND. ( SCHUMAX .GT. 0 ) ) .OR. &
+              ( ( EDMFMAX .GT. 0 ) .AND. ( model_config_rec%ishallow .GT. 0 ) ) ) THEN
+            wrf_err_message = '--- ERROR: bl_mynn_edmf > 0 requires both shcu_physics=0 & ishallow=0' 
+            CALL wrf_message(wrf_err_message)
+            wrf_err_message = 'when using MYNN PBL, by default bl_mynn_edmf is turned on'
+            CALL wrf_message(wrf_err_message)
+            wrf_err_message = 'Modify namelist.input so that shcu_physics nor ishallow is used when bl_mynn_edmf is turned on'
+            CALL wrf_debug ( 0, TRIM( wrf_err_message ) )
+            count_fatal_error = count_fatal_error + 1
          END IF
```

Comment:
This will not allow the domain that doesn't use MYNN/EDMF to use another shallow scheme. Is this more restrictive than what we want?